tutorials/gm-world.md

```diff
@@ -626,63 +626,7 @@ pagination:
   total: "0"
 ```
 
-## Part Two
-
-### 🛑 Stopping our local devnet rollup {#stop-your-local-devnet}
-
-First, you will need to stop your current devnet version of the gm rollup.
-Follow instructions to stop the container below.
-
-Find the Container ID that is running by using the command:
-
-```bash
-docker ps
-```
-
-Then stop the container:
-
-```bash
-docker stop CONTAINER_ID_or_NAME
-```
-
-#### 🟢 Restarting your container {#restart-your-container}
-
-Optionally, if you'd like to restart your container, you can use the `docker start` command followed by the container ID or name.
-
-You can obtain the container ID or name of a stopped container using the `docker ps -a` command, which will list all containers (running and stopped) and their details. For example:
-
-```bash
-docker ps -a
-```
-
-This will give you an output similar to this:
-
-```bash
-CONTAINER ID   IMAGE                                            COMMAND            CREATED         STATUS         PORTS                                                                                                                         NAMES
-d9af68de54e4   ghcr.io/rollkit/local-celestia-devnet:v0.9.1   "/entrypoint.sh"   5 minutes ago   Up 2 minutes   1317/tcp, 9090/tcp, 0.0.0.0:26657->26657/tcp, :::26657->26657/tcp, 26656/tcp, 0.0.0.0:26659->26659/tcp, :::26659->26659/tcp   musing_matsumoto
-```
-
-In this example, you can restart the container using either its container ID (`d9af68de54e4`) or name (`musing_matsumoto`). To restart the container, run:
-
-```bash
-docker start d9af68de54e4
-```
-
-or
-
-```bash
-docker start musing_matsumoto
-```
-
-#### 🗑️ Removing your container {#removing-your-container}
-
-If you ever would like to remove the container, you can use the `docker rm` command followed by the container ID or name.
-
-Here is an example:
-
-```bash
-docker rm CONTAINER_ID_or_NAME
-```
+## Part two
```

Comment:
Is there some other place where we could add this information. Is it relevant for the developer? Maybe have a separate localcelestia-devnet section or adding it to the Read me ? Thinking out loud , wdyt

Reply:
good thinking, thanks for review. maybe it's time to split this up into 3 tutorials?

Reply:
but i think this could be v1

Reply:
Yeah, I think localcelestia-devnet could have its own tutorial. And Arabica / mocha / mainnet net could have 3 seperators? Like [here](https://docs.celestia.org/nodes/light-node#optional-run-the-light-node-with-a-custom-key). Ideally it could work with 3 flags so the developer only needs to change the name of the network. Something we might want to consider in a follow-up and improve our DevUX

Reply:
As more DA-Layers will come around, more sites will be added so we might  need to find a way to make this more scalable

Reply:
We can resolve this. I created 2 Follow issues. #268 #267